src/types.ts

```diff
@@ -31,27 +31,31 @@ export const enum State {
  * @readonly
  * @enum {string}
  */
-export enum BotKind {
-  Awesomium = 'awesomium',
-  Cef = 'cef',
-  CefSharp = 'cefsharp',
-  CoachJS = 'coachjs',
-  Electron = 'electron',
-  FMiner = 'fminer',
-  Geb = 'geb',
-  NightmareJS = 'nightmarejs',
-  Phantomas = 'phantomas',
-  PhantomJS = 'phantomjs',
-  Rhino = 'rhino',
-  Selenium = 'selenium',
-  Sequentum = 'sequentum',
-  SlimerJS = 'slimerjs',
-  WebDriverIO = 'webdriverio',
-
-  WebDriver = 'webdriver',
-  HeadlessChrome = 'headless_chrome',
-  Unknown = 'unknown',
-}
+export const BotKind = {
+  // Object is used instead of Typescript enum to avoid emitting IIFE which might be affected by further tree-shaking.
```

Comment:
Put it here because it's maintainer-faced info, don't think we need it in the JSDoc annotation